### ISSUE TITLE
关于yolov5 batchnorm 层 eps参数的问题

### ISSUE THREAD

**Issue by jmydurant** [wang-xinyu/tensorrtx#209]
非常感谢您的这份工作，想咨询以下。YOLOv5是pytorch实现的，但是他用的bn层貌似是默认的。我看你在具体层实现的时候分别用了1e-4 和 1e-3 与 pytorch默认的1e-5不同，是出于什么考虑么。还是说我看漏了。
能否抽出时间解答一下这个疑惑。多谢多谢

**Comment by wang-xinyu** [comment]
参考这个issue
https://github.com/wang-xinyu/tensorrtx/issues/183

**Comment by jmydurant** [comment]
> 参考这个issue
> #183

你好，我看了下 yolov5 的 train.py 其中有一段

# SyncBatchNorm
    if opt.sync_bn and cuda and rank != -1:
        model = torch.nn.SyncBatchNorm.convert_sync_batchnorm(model).to(device)
        logger.info('Using SyncBatchNorm()')

但是在pytorch 1.6版本中 找到 SyncBatchNorm 代码部分的注释

Args:
        num_features: :math:`C` from an expected input of size
            :math:`(N, C, +)`
        eps: a value added to the denominator for numerical stability.
            Default: ``1e-5``
        momentum: the value used for the running_mean and running_var
            computation. Can be set to ``None`` for cumulative moving average
            (i.e. simple average). Default: 0.1

其中 eps 默认值还是 1e-5

**Comment by wang-xinyu** [comment]
那print model看下，实际好像是1e-3

**Comment by pengboc** [comment]
yolo.py 的 100 行调用的初始化权重函数 `initialize_weights(self)` 中，将 BN 中的 eps 设为 1e-3
